.github/workflows/ios.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         with:
           version: ${{ env.INSTALL_QT_VERSION }}
           target: desktop
-          cache: false
+          cache: true
```

Comment:
To be safe I'd add a [cache-key-prefix](https://github.com/jurplel/install-qt-action#cache-key-prefix)

Reply:
@nirvn actually, I am not so sure. It seems the autogenerated cache key is quite good (contains also components etc) are we sure want to manually manage?

Reply:
@m-kuhn , we definitely had issues with Android arch types, with day armv8 showing up on armv7, etc. That's why we had disabled cache altogether to begin with.

Reply:
Ouch. That's something that could/should be fixed upstream.